dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -40,10 +51,123 @@ def get_unique_dimension_rows(self):
         """Get distinct combinations of remapped dimensions, including id.
         Check each col in combination for null value."""
         dim_table = self._remap_dimension_columns(self._load_data_lookup).distinct()
-        self._check_null_value_in_unique_dimension_rows(dim_table)
-
+        check_null_value_in_unique_dimension_rows(dim_table)
         return dim_table
 
+    def make_project_dataframe(
+        self, load_data_df=None, lookup_df=None, query_context: QueryContext = None
+    ):
+        ld_df = load_data_df or self._load_data
```

Comment:
what happens if both are available?

Reply:
load_data_df takes precedence. self._load_data is always available.